apps/examples/testcase/le_tc/filesystem/fs_main.c

```diff
@@ -1662,6 +1662,71 @@ static void libc_stdio_fread_tc(void)
 }
 
 /**
+* @testcase         libc_stdio_freopen_tc
+* @brief            Open file by freopen
+* @scenario         Open file
+* @apicovered       freopen
+* @precondition     NA
+* @postcondition    NA
+*/
+static void libc_stdio_freopen_tc(void)
+{
+	FILE *fp;
+	char *filename = VFS_FILE_PATH;
+
+	fp = freopen(NULL, "w", NULL);
+	TC_ASSERT_EQ("freopen", fp, NULL);
+
```

Comment:
This should be TC_ASSERT_EQ_CLEANUP("freopen", fp, NULL, fclose(fp)); also.
And please move it down to negative cases.